packages/marketplace/src/components/pages/client.tsx

```diff
@@ -1,10 +1,9 @@
 import * as React from 'react'
```

Comment:
Pls move this page and test file to <page-name> folder instead of let it in pages.
![image](https://user-images.githubusercontent.com/22660833/82865445-ecf68380-9f50-11ea-994c-0128d962d52c.png)